flocker/testtools/__init__.py

```diff
@@ -201,8 +201,14 @@ def loop_until(predicate):
     :return: A ``Deferred`` firing with the first ``Truthy`` response from
         ``predicate``.
     """
-    msg("Looping on %s (%s:%s)" % (predicate, getfile(predicate),
-                                   getsourcelines(predicate)[1]))
+    try:
+        msg("Looping on %s (%s:%s)" % (predicate, getfile(predicate),
+                                       getsourcelines(predicate)[1]))
+    except IOError:
```

Comment:
I saw this exception raised but have yet to test this code. I'll do it on Monday but the docs are still ready for review.